assets/src/stories-editor/components/custom-video-block-edit.js

```diff
@@ -101,10 +103,19 @@ class CustomVideoBlockEdit extends Component {
 			getContentLengthFromUrl( src ).then( ( videoSize ) => {
 				this.setState( { videoSize } );
 			} );
+
+			if ( ! poster ) {
+				if ( videoFeaturedImage ) {
+					setAttributes( { poster: videoFeaturedImage.source_url } );
+				} else {
+					uploadVideoFrame( src );
```

Comment:
Would be good to display a thumbnail, too, it's unclear right now which image is being used, it just shows "Replace Image".

Reply:
True. This could be more aligned with how the page background poster is displayed.